agent/config/parse_windows.go

```diff
@@ -34,14 +34,24 @@ const (
 // parseGMSACapability is used to determine if gMSA support can be enabled
 func parseGMSACapability() bool {
 	envStatus := utils.ParseBool(os.Getenv("ECS_GMSA_SUPPORTED"), true)
+	return checkDomainJoin(envStatus)
+}
+
+// parseFSxWindowsFileServerCapability is used to determine if fsxWindowsFileServer support can be enabled
+func parseFSxWindowsFileServerCapability() bool {
+	envStatus := utils.ParseBool(os.Getenv("ECS_FSX_WINDOWS_FILE_SERVER_SUPPORTED"), true)
+	return checkDomainJoin(envStatus)
+}
+
+func checkDomainJoin(envStatus bool) bool {
```

Comment:
Can you rename the method and pass in the option for the env override?

Reply:
will do